chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
So the idea is to populate the list with all available style locales in order of their language code. Plus, copy the fallback (i.e. Zotero) locale to the top of the list. If there is a preferred locale, copy that to the top as well and select it. Is that right?

What exactly is the intention with the "preferred" locale here? From what I can tell, that's the most-recently-used locale, correct?

In either case, I think we should add a separator between the general list of locales and the fallback/preferred locales. This would make the list look nicer if, say, "Afrikaans" is the fallback language (you would have two Afrikaans in a row). And no, I don't think we should (re)move the preferred/fallback language from the general list.